sound/soc/sof/topology.c

```diff
@@ -2947,9 +2947,12 @@ static int sof_link_dmic_load(struct snd_soc_component *scomp, int index,
 			config->dmic.pdm[j].skew);
 	}
 
+	/*
+	 * this takes care of backwards compatible handling of fifo_bits_b.
+	 * It is deprecated since 3.0.1.
+	 */
 	if (SOF_ABI_VER(v->major, v->minor, v->micro) < SOF_ABI_VER(3, 0, 1)) {
-		/* this takes care of backwards compatible handling of fifo_bits_b */
-		config->dmic.reserved_2 = config->dmic.fifo_bits;
+		config->dmic.fifo_bits_b = config->dmic.fifo_bits;
```

Comment:
some people prefer to use curly braces if there is more than one *line* in the block, even though it's just one command, e.g.
```
if (ret < 0) {
	/* error */
	return ret;
}
```
or
```
if (ret < 0) {
	dev_err(dev, "error %d\n",
			ret);
}
```
but here we have one command on one line... Do we still need braces?..

Reply:
@lyakh I'd say it would be pretty confusing and error-prone given the comment. So I for one would recommend keeping the braces as long as the comment exists.

Reply:
Yep, I'll change. Maybe those were used because of the comment.

Reply:
Done!